test/cli.js

```diff
@@ -15,6 +15,11 @@ describe('node:cli', function () {
     assert.fileContent('lib/cli.js', 'import meow from \'meow\'');
   });
 
+  it('Contains package name references', function() {
+    assert.noFileContent('lib/cli.js', 'var = require(\'./\');');
```

Comment:
let's assert the correct behavior. Negative assertion are not useful to prove something is working correctly.

As you can see, here you have two assertions and both are passing. But one of them wouldn't fail even before your fix.

This mean you need to add two tests, one for the normal mode and one for the Babel mode.